domain/service/service.go

```diff
@@ -86,6 +86,7 @@ type Service struct {
 	LogConfigs        []servicedefinition.LogConfig
 	Snapshot          servicedefinition.SnapshotCommands
 	Runs              map[string]string
```

Comment:
Should probably add a FIXME or TODO comment here to remind us that Runs should be removed in some future release. Not sure what that release should be, but eventually we should clean it up.